.github/workflows/quick-start.yml

```diff
@@ -0,0 +1,47 @@
+name: quick-start
+
+on: 
+ workflow_dispatch:
+
+jobs:
+  test-quick-start:
+    name: "Run quick start test"
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    permissions:
+      contents: read
+    strategy:
+      matrix:
+        KUBERNETES_VERSION: ["1.26.3"]
```

Comment:
should we also specify `GATEKEEPER_VERSION`?

Reply:
thanks , GATEKEEPER_VERSION removed.